src/ex6/e2e/cypress/e2e/add-todo.cy.js

```diff
@@ -0,0 +1,17 @@
+describe("Add Todo Action", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000");
+  });
+
+  it("Should add a new todo", () => {
+    // TODO: fill this test
+
+    cy.get(".TodoList_title__XX0Ta").should("be.visible");
+    cy.wait(1000);
+    cy.get(".TodosHeader_inputHeader__fWyos  ").type("do cypress testing!");
+    cy.get(".TodosHeader_addBtn__YiJSC").click();
+    cy.contains("do cypress testing!");
```

Comment:
you dont actually test that the item was created here, testing that the text exists somewhere in the html is not enough